### DISCUSSION TITLE
Is an optimized pagination algorithm used here?

### DISCUSSION THREAD

**Discussion by AlexRMU** [prisma/prisma#23699] [Q&A] [answered]
That is, is the deferred join used here or something similar?
https://planetscale.com/blog/mysql-pagination#deferred-joins-for-faster-offset-limit-pagination

**Comment by ludralph** [comment, marked as answer]
Hi @AlexRMU 👋 

Thank you for raising this question.

Prisma Client does not use deferred joins for pagination. If you would love to see this added, you can open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) with a concrete example of a Prisma Client query where this would apply so our engineers could take a look.
